src/textual_summary/data/operation_ranges.js

```diff
@@ -0,0 +1,15 @@
+export const operationRangesData = {
+  title: 'Operation Ranges',
+  items: [
+    {
+      label: 'CPU',
```

Comment:
should these strings be extracted? or are they used as props somewhere else?

Reply:
This is test data. It  is used in stories and tests.